qiskit/transpiler/passes/optimization/consolidate_blocks.py

```diff
@@ -195,38 +130,15 @@ def _handle_control_flow_ops(self, dag):
         pass_manager = PassManager()
         if "run_list" in self.property_set:
             pass_manager.append(Collect1qRuns())
-        if "block_list" in self.property_set:
```

Comment:
It can be, that's arguably the normal invocation of the pass. My thinking here was that we'll implicitly run the equivalent of `Collect2qBlocks` on the control flow block regardless of the property set if we don't specify it. That's the new feature this PR adds to the pass. So we don't need to manually populate the property set with `Collect2qBlocks` unless the `run_list` is set because populating that will preclude the 2q blocks.